### PR TITLE
[zh] add a device plugin example

### PR DESCRIPTION
**Pull request by liuyuanzhi-cn**
add a device plugin example

### CODE REVIEW

**Comment by ydFu** [comment]
@liuyuanzhi-cn ,thank you for your contribution.

You must sign the CLA for teams of kubernetes/website to do code reiview.

Please click on the red button `NOT COVERED` line in the first comment.

If you don't know how to do it, here is[the CLA tutorial](https://git.k8s.io/community/CLA.md).

File: content/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md [hunk 1]
```diff
@@ -818,6 +818,7 @@ Here are some examples of device plugin implementations:
 * [Solarflare 设备插件](https://github.com/vikaschoudhary16/sfc-device-plugin)
 * [SR-IOV 网络设备插件](https://github.com/intel/sriov-network-device-plugin)
 * [Xilinx FPGA 设备插件](https://github.com/Xilinx/FPGA_as_a_Service/tree/master/k8s-device-plugin)
+* [linux的通用设备插件](https://github.com/squat/generic-device-plugin)
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md:821]:
```suggestion
* Linux 的[通用设备插件](https://github.com/squat/generic-device-plugin)
```

**Comment by T-Lakshmi** [comment]
@liuyuanzhi-cn,
As per community guidelines, we work on one PR for one language. Multiple languages in a single PR is not allowed.
i suggest you to break this PR into two(en,zh).

**Comment by T-Lakshmi** [comment]
@liuyuanzhi-cn,
You may continue this PR with **zh** changes as you already received review feedback and can open a separate PR for **en** language.

In future, best practice is to first merge PR with en language and after sync the remaining languages with en.

**Comment by T-Lakshmi** [comment]
/remove-language en

@liuyuanzhi-cn,
Please open a separate PR for website repo primary content(i.e., English language).

**Comment by liuyuanzhi-cn** [comment]
> /remove-language en
> 
> @liuyuanzhi-cn, Please open a separate PR for website repo primary content(i.e., English language).

pr has been created [here](https://github.com/kubernetes/website/pull/42194)

**Comment by tengqm** [comment]
/hold

Thank you for proposing this fix. However, we will not merge partial fixes as per stated in the [guideline](https://kubernetes.io/zh/docs/contribute/localization_zh/#%E6%9B%B4%E6%96%B0%E8%AF%91%E6%96%87). To find out what are the upstream changes to be synced, do this **on the main branch**:

```shell
./scripts/lsync.sh content/zh-cn/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins.md
```

**Comment by tengqm** [comment]
English version change closed, zh version is a partial sync...
No response from author for two months.
/close

**Pull request closed without merging**